internal/monitor/integration_steps_test.go

```diff
@@ -75,6 +76,7 @@ var wordsToNumberMap = map[string]float64{
 var failureToScriptMap = map[string]string{
 	"interfacedown": "bounce.ip",
 	"reboot":        "reboot.node",
+	"kubeletdown":   "bounce.kubelet",
 }
```

Comment:
Was it really this simple?!?

Reply:
Yes, get from this map will return the right script to call, we all needed to pass is failureType